### PR TITLE
feat(create event): add and create new event feature

### LINKED ISSUE #20: CreateEvent
**Task title**
CreateEvent

**Task description**
create event allow the user to add an event to the collection of events existing in the database

**Subtasks**
prompt user input
submit user input
store and post input
instant change display of the event

**Screenshots**
later

**Link to the component on Figma**
later

### PR DESCRIPTION
**Pull request by Khalil-NOUI**
provide an input form that allow the user to add new event with all of its informations

BREAKING CHANGE: add new event and display it in th events page

"closes #20"

# Related Issue

- Resolve #20

### CODE REVIEW

File: src/components/creatEventPageComponents/CreateEvent.jsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                    type='button'
+                    className='text-gray-600 bg-transparent hover:bg-gray-200 hover:text-gray-900 rounded-lg text-sm p-1.5 ml-auto inline-flex items-center'
+                    data-modal-toggle='Event-modal'
+                >
+                    <svg
+                        className='w-5 h-5'
+                        fill='currentColor'
+                        viewBox='0 0 20 20'
+                        xmlns='http://www.w3.org/2000/svg'
+                    >
```
Comment (HachemBouhadede) [inline src/components/creatEventPageComponents/CreateEvent.jsx:35]: i'm not sure what this svg is for ! let me know during today meeting

File: src/components/creatEventPageComponents/CreateEvent.jsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                        <path
+                            fillRule='evenodd'
+                            d='M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z'
+                            clipRule='evenodd'
+                        ></path>
```
Comment (HachemBouhadede) [inline src/components/creatEventPageComponents/CreateEvent.jsx:40]: same as the svg not sure what this path is for

File: src/components/creatEventPageComponents/CreateEvent.jsx
Comment (HachemBouhadede) [inline src/components/creatEventPageComponents/CreateEvent.jsx:1]: funnctionality wise everythign seems good, although there are a few things i didn't understand their purpse, for some reason i can't addDoc from firebase/firestore on github will try to check it manually

File: src/pages/createEvent/index.jsx
Comment (HachemBouhadede) [inline src/pages/createEvent/index.jsx:1]: you could"ve implement the creatEvent.jsx here i believe ! but i'm sure you have your reasons

**Review by HachemBouhadede — APPROVED** [review]

**Comment by HachemBouhadede** [comment]
over all well done khalil ! gj buddy <3

File: src/components/creatEventPageComponents/CreateEvent.jsx (hunk 2 above)
Comment (Khalil-NOUI) [reply src/components/creatEventPageComponents/CreateEvent.jsx:40]: i totally missed it, it's feature of the template provided, which definetly can be ignored for our purpose of usage, thank you for this deep observation

File: src/components/creatEventPageComponents/CreateEvent.jsx (hunk 1 above)
Comment (Khalil-NOUI) [reply src/components/creatEventPageComponents/CreateEvent.jsx:35]: they are extra features (X sign to close window etc..)  of the template provided, in our purpose of usage they can totally be removed

**Comment by HachemBouhadede** [comment]
great work on fixing the deployment issue !

**Pull request merged**